eng/Version.Details.xml

```diff
@@ -11,302 +11,302 @@
     </Dependency>
     <Dependency Name="System.ServiceProcess.ServiceController" Version="5.0.0">
       <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>cf258a14b70ad9069470a108f13765e0e5988f51</Sha>
+      <Sha>bfc49945c0bedeffe01bb5d6f3c217dad207d0d8</Sha>
     </Dependency>
     <Dependency Name="System.Text.Encodings.Web" Version="5.0.0">
       <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>cf258a14b70ad9069470a108f13765e0e5988f51</Sha>
+      <Sha>bfc49945c0bedeffe01bb5d6f3c217dad207d0d8</Sha>
     </Dependency>
     <Dependency Name="System.Text.Json" Version="5.0.0">
       <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>cf258a14b70ad9069470a108f13765e0e5988f51</Sha>
+      <Sha>bfc49945c0bedeffe01bb5d6f3c217dad207d0d8</Sha>
     </Dependency>
     <Dependency Name="System.Threading.Channels" Version="5.0.0">
       <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>cf258a14b70ad9069470a108f13765e0e5988f51</Sha>
+      <Sha>bfc49945c0bedeffe01bb5d6f3c217dad207d0d8</Sha>
     </Dependency>
     <Dependency Name="System.Windows.Extensions" Version="5.0.0">
       <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>cf258a14b70ad9069470a108f13765e0e5988f51</Sha>
+      <Sha>bfc49945c0bedeffe01bb5d6f3c217dad207d0d8</Sha>
     </Dependency>
     <Dependency Name="Microsoft.Extensions.DependencyModel" Version="5.0.0">
       <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>cf258a14b70ad9069470a108f13765e0e5988f51</Sha>
+      <Sha>bfc49945c0bedeffe01bb5d6f3c217dad207d0d8</Sha>
     </Dependency>
     <Dependency Name="Microsoft.NETCore.App.Ref" Version="5.0.0">
       <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>cf258a14b70ad9069470a108f13765e0e5988f51</Sha>
+      <Sha>bfc49945c0bedeffe01bb5d6f3c217dad207d0d8</Sha>
     </Dependency>
     <!--
          Win-x64 is used here because we have picked an arbitrary runtime identifier to flow the version of the latest NETCore.App runtime.
          All Runtime.$rid packages should have the same version.
     -->
-    <Dependency Name="Microsoft.NETCore.App.Runtime.win-x64" Version="5.0.0">
+    <Dependency Name="Microsoft.NETCore.App.Runtime.win-x64" Version="5.0.1">
       <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>cf258a14b70ad9069470a108f13765e0e5988f51</Sha>
+      <Sha>bed28f348ccd4bffbac619eb5c55c98836a3ef4b</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.NETCore.App.Internal" Version="5.0.0-rtm.20519.4">
+    <Dependency Name="Microsoft.NETCore.App.Internal" Version="5.0.1-servicing.20566.8">
       <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>cf258a14b70ad9069470a108f13765e0e5988f51</Sha>
+      <Sha>bed28f348ccd4bffbac619eb5c55c98836a3ef4b</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.NETCore.BrowserDebugHost.Transport" Version="5.0.0-rtm.20519.4">
+    <Dependency Name="Microsoft.NETCore.BrowserDebugHost.Transport" Version="5.0.0-servicing.20566.8">
```

Comment:
@jkoritzinsky this PR scares me when thinking about #27737 if this package version does _not_ always match that of the dotnet/runtime archives, we can't use it in 6.0.0. Thoughts❔

@Anipik @ericstj the other possibility here (where the numbers match the update just above) is there's a slight bug in the version settings for this package.

Reply:
This package version definitely should match the rest of the outputs from the dotnet/runtime repo. If it isn't just a fluke then I'd be 100% ok switching which non-shipping package we use or introducing a new one just for this usage to ensure that we don't break things terribly.

Reply:
Let's wait to hear from @Anipik or @ericstj before we decide what action to take

Reply:
/ping @Anipik and @ericstj

Reply:
I believe the issue with the versioning here was fixed by https://github.com/dotnet/runtime/pull/44832 but I'm not sure.

Reply:
Yup, based on https://dev.azure.com/dnceng/internal/_build/results?buildId=890966&view=artifacts&type=publishedArtifacts the problem is fixed❕

Reply:
#27916 also corrects the version in this repo: https://github.com/dotnet/aspnetcore/pull/27916/files#diff-fb62e94a1d6f29f863e3d0a22aa38269f6cd1d7f03b109dc06e2cbf2548b86d3L300-R300